### PR TITLE
Make it possible to type a time in the sleep timer

### PR DESCRIPTION
**Pull request by The-Fireplace**
As a user, I want to be able to enter a specific amount of time for the sleep timer to count down to without dragging the slider.

This PR addresses that by making it so the time is now a text box that retains the previous behavior of showing the time and adjusting it with the slider, and also allows the user to type the time, and adjusts the slider accordingly if they do.

https://user-images.githubusercontent.com/5914284/109899727-e8879580-7c5b-11eb-90d1-9442e87d6490.mp4

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -199,6 +199,7 @@
     <string name="cancel_current_timer">Cancel current timer</string>
     <string name="sleep_timer_canceled">Sleep timer canceled.</string>
     <string name="sleep_timer_set">Sleep timer set for %d minutes from now.</string>
+    <string name="time_unit_min">min</string>
```
Comment (soncaokim) [inline app/src/main/res/values/strings.xml:202]: Please use the full word "minute" to avoid misunderstanding.

**Review by soncaokim — APPROVED** [review]
lgtm

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (The-Fireplace) [reply app/src/main/res/values/strings.xml:202]: I left it as "min" because that's what it said there before.

Comment (soncaokim) [reply app/src/main/res/values/strings.xml:202]: I mean to rename `time_unit_min` to `time_unit_minute`.

**Pull request merged**